lib/trento/software_updates/enums/software_updates_health.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Trento.SoftwareUpdates.Enums.SoftwareUpdatesHealth do
+  @moduledoc """
+  Type that represents the possible health values for the software updates discovery process.
+  """
+
+  use Trento.Support.Enum, values: [:passing, :warning, :critical, :unknown, :not_set]
```

Comment:
Maybe `unset` is more appropriate that `not_set`.
What does our anglo native @jamie-suse say hehe ?